### ISSUE TITLE
Station icons missing - v2.36.0

### ISSUE THREAD

**Issue by dcadams** [expectingtofly/LMS_BBC_Sounds_Plugin#173]
Not totally sure that this plugin is the issue but I have been keeping it updated with the latest recently and this issue has only just happened.

Running both BBC Sounds and Global Player plugins in Daphile v23.01. The station icons are missing in FF and Chrome. 
![image](https://github.com/expectingtofly/LMS_BBC_Sounds_Plugin/assets/1840555/bf2b2034-d634-4a43-a454-7c8cb2686731)
![image](https://github.com/expectingtofly/LMS_BBC_Sounds_Plugin/assets/1840555/2d5eb6e2-5b6f-46c2-a30d-43fb866107d5)

The stations that are not of either of these 2 plugins appear as normal.

**Comment by expectingtofly** [comment]
Sorry for the delay in replying, I've been away.

The icons are associated with a favourite when you add the favourite in LMS. So not really a plugin issue.  
I'm not familiar with daphile. What version of LMS are you using?

Were the icons missing when you added the favourite, or have they recently disappeared?
Are the icons there when you browse through the BBC Sounds menu?

**Comment by dcadams** [comment]
Thank you for the reply @expectingtofly 

Running the very latest version of Daphile (v23.01) which, from the changelog, includes:
LMS update to Jan 4th 2023 version of 8.3 branch (git commit: 353c504)

The icons were there when I added the favourites which may have been over a year ago. Just in the last couple of weeks they have disappeared. When I browse the BBC Sounds menu the icons are also missing but they were definitely there originally.

![image](https://github.com/expectingtofly/LMS_BBC_Sounds_Plugin/assets/1840555/76f4afee-5a16-4b4f-a9b2-e3a222a18143)

**Comment by dcadams** [comment]
A bit more info. Here's a snippet from the LMS log:
[24-03-29 00:00:33.4279] main::init (377) Starting Logitech Media Server (v8.3.1, git353c504, Daphile 23.01-x86_64) perl 5.036000 - x86_64-linux-thread-multi
[24-03-29 00:00:38.1262] Slim::Formats::XML::gotErrorViaHTTP (343) Error: getting https://www.mysqueezebox.com/api/sounds/v1/opml
404 Not Found
[24-03-29 00:00:38.1269] Slim::Plugin::Sounds::Plugin::_gotSoundsError (82) Error: Unable to cache Sounds & Effects menu from SN: 404 Not Found
[24-03-29 00:29:36.1776] Plugins::GlobalPlayerUK::ProtocolHandler::__ANON__ (674) error: Connect timed out: Resource temporarily unavailable
[24-03-29 00:34:42.7998] Plugins::GlobalPlayerUK::ProtocolHandler::__ANON__ (674) error: Connect timed out: Resource temporarily unavailable

Is the issue caused by the shutdown of MySqueezebox.com servers? (https://www.mysqueezebox.com/index/Home)

**Comment by expectingtofly** [comment]
@dcadams 
> Is the issue caused by the shutdown of MySqueezebox.com servers? (https://www.mysqueezebox.com/index/Home)

Could be,  in LMS settings->Advanced->Performance->Artwork Resizing.   
Have you got that set to MySqueezebox ?
![image](https://github.com/expectingtofly/LMS_BBC_Sounds_Plugin/assets/73394021/45e136cd-05a9-4961-9f6c-cca4d0c0c85c)



>
> Running the very latest version of Daphile (v23.01) which, from the changelog, includes: LMS update to Jan 4th 2023 version of 8.3 branch (git commit: 353c504)
> 
Is there a way to get that updated to the non-Mysqueezebox version of LMS 8.5 on Daphile ?

**Comment by dcadams** [comment]
I did have that setting set to MySqueezebox. Having changed it to LMS the icons have now reappeared. Thank you for that as I did not know about that setting.

As you say, the better solution would be if Daphile were updated but the developer hasn't surfaced in over a year. If it was OS I'd have a look at it myself but unfortunately it isn't.

Anyway, thank you for the help and also for creating and maintaining these great plugins.
